cmd/events.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/juju/ansiterm"
+	"github.com/urfave/cli"
+
+	manifold "github.com/manifoldco/go-manifold"
+	"github.com/manifoldco/go-manifold/events"
+	"github.com/manifoldco/manifold-cli/api"
+	"github.com/manifoldco/manifold-cli/color"
+	"github.com/manifoldco/manifold-cli/middleware"
+	"github.com/manifoldco/manifold-cli/prompts"
+)
+
+func init() {
+	eventsCmd := cli.Command{
+		Name:     "events",
+		Usage:    "Review the most recent activity of your account",
+		Category: "ADMINISTRATIVE",
```

Comment:
Should we have some sort of.. pagination? limiting flags.

Cause once you get a lot of events, this is going to create a lot of scrollback.

Reply:
I was thinking about limiting on the server for now., ie: latest 10 per query. I'm not sure if your `db` implementation supports limit / offset already.

Reply:
Is that only planned? or is it happening.

If it's just planned but not prioritized yet, we should add a simple limit/offset with flags here.
Otherwise ok

Reply:
Added both flags